cmd/tether/ops.go

```diff
@@ -36,9 +42,17 @@ func (t *operations) Setup() error {
 }
 
 func (t *operations) Cleanup() error {
+	if err := t.BaseOperations.Cleanup(); err != nil {
```

Comment:
`return t.BaseOperations.Cleanup()`
but I don't think you need this at all if `t` is embedding `BaseOperations`